### PR TITLE
Update notifications.md

### PR DESCRIPTION
**Pull request by HennaAbbas**
Removed documentation about Slack integration and added a valid config that uses contexts instead of the webhook key.

# Description
Removed documentation about slack integration because the Slack integration is no longer supported. Also added a valid cofig for the currently supported slack orb ( which no longer uses the webhook key) 

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: jekyll/_cci2/notifications.md [hunk 1]
```diff
@@ -90,34 +90,39 @@ Before integrating an orb into your configuration, you will need to perform two
 Using the [CircleCI Slack Orb](https://circleci.com/developer/orbs/orb/circleci/slack), you can integrate and customize Slack notifications directly from your configuration file. The following config is an example of notifying a Slack channel with a custom message:
```
Comment (rosieyohannan) [inline jekyll/_cci2/notifications.md:90]:
```suggestion
Using the [CircleCI Slack orb](https://circleci.com/developer/orbs/orb/circleci/slack), you can integrate and customize Slack notifications directly from your configuration file. The following config is an example of notifying a Slack channel with a custom message:
```

File: jekyll/_cci2/notifications.md [hunk 2]
```diff
@@ -90,34 +90,39 @@ Before integrating an orb into your configuration, you will need to perform two
...
+      - notify:
+          context: slack-secrets      
 ```
 
-To get started with enabling notifications in Slack *for a specific project*, head to the *Settings* page for your project. Click on *Slack Integration* in the sidebar and follow the steps to setup Slack.
-
-![]({{ site.baseurl }}/assets/img/docs/notification-page-slack.png)
-
-It is also possible to use the Slack Orb to provide other types of notifications, including notifying a slack channel of a pending approval or sending a status alert at the end of a job based on success or failure. To view such usage examples, consult the [CircleCI Slack Orb page](https://circleci.com/developer/orbs/orb/circleci/slack).
+CircleCI's Slack Orb can be used to provide other types of notifications, including notifying a slack channel of a pending approval or sending a status alert at the end of a job based on success or failure. To view such usage examples, consult the [CircleCI Slack Orb page](https://circleci.com/developer/orbs/orb/circleci/slack).
```
Comment (rosieyohannan) [inline jekyll/_cci2/notifications.md:125]:
```suggestion
CircleCI's Slack orb can be used to provide other types of notifications, including notifying a slack channel of a pending approval or sending a status alert at the end of a job based on success or failure. For more information and to view usage examples, see the [CircleCI Slack orb page](https://circleci.com/developer/orbs/orb/circleci/slack).
```

**Comment by rosieyohannan** [comment]
@HennaAbbas Thanks so much for updating this page for us! Just a couple of minor changes and then good to go :-D

**Comment by HennaAbbas** [comment]
@rosieyohannan I went ahead and committed those changes. Please let me know if there is anything else on my end. Thanks!

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**